contrib/opencensus-ext-stackdriver/tests/test_stackdriver_stats.py

```diff
@@ -840,6 +852,13 @@ def test_create_timeseries(self, monitor_resource_mock):
         )
         self.assertEqual(value.distribution_value, expected_distb)
 
+        start_time_pb = time_series.points[0].interval.start_time.timestamp_pb()
+        end_time_pb = time_series.points[0].interval.end_time.timestamp_pb()
+        self.assertEqual(start_time_pb.seconds, 1545699663)
+        self.assertEqual(start_time_pb.nanos, 4053)
+        self.assertEqual(end_time_pb.seconds, 1545699723)
+        self.assertEqual(end_time_pb.nanos, 4053)
```

Comment:
Added this test in 68bda19c5e68429448fd2fbb9dadb7307ca87313 (before other changes in this PR) to test this works with new protoplus timestamps.